cargo-bpf/src/build.rs

```diff
@@ -121,6 +150,23 @@ pub fn build_program(
     Ok(())
 }
 
+fn get_opt_executable() -> Result<String, Error> {
+    for llc in vec!["opt".into(), env::var("OPT").unwrap_or("opt-9".into())].drain(..) {
```

Comment:
This copy-pasta from the function below, we should probably keep one LLVM version check.

Reply:
I just copy/pasted since hopefully in the next few days that code is going to be deleted entirely once I figure out the passes to configure in llvm to compile directly from cargo-bpf without having to call out to opt/llc.

If you still want me to rework it, I'm happy to do that.

Reply:
I'm happy with that approach.